frontend/src/app/tasks/tasks.component.spec.ts

```diff
@@ -32,6 +33,44 @@ class TestComponent {
   }
 }
 
+// Beware: this tester assumes that the component has one and only has one task
+class TasksComponentTester extends ComponentTester<TestComponent> {
+  constructor(fixture: ComponentFixture<TestComponent>) {
+    super(fixture);
+  }
+
+  get text() {
+    return this.element('.task-item').textContent;
+  }
+
+  get addSpentTimeLink() {
+    return this.element('.add-spent-time-link') as TestHtmlElement<HTMLElement>;
```

Comment:
I don't know if it's better, but I would have tried `this.element<HTMLElement>()`

Reply:
I tried a whole lot of things, including that. The less cumbersome way I found was the one I finally picked. I also think it's easier to use for those who aren't too versed into generics.